praw/exceptions.py

```diff
@@ -172,6 +172,17 @@ def __init__(self):
         )
 
 
+class InvalidMedia(ClientException):
+    """Indicate exceptions where invalid/corrupted media is used."""
+
+    def __init__(self):
+        """Instantize the class."""
+        super().__init__(
+            "The media you have tried to upload is invalid. Check to see that"
+            " you can open the media file on your local system."
```

Comment:
I think that perhaps we should rename the exception and its message. Just because the type of the update is `"failed"` does not necessarily mean that the media was bad — it could have failed due to some server error, for instance. 

I think it might be better to have a `MediaPostFailed` exception, or something like that. Then, in the text we can say something like "Maybe the media you tried to upload is invalid. Check to see…"